cropduster/templatetags/cropduster_tags.py

```diff
@@ -42,29 +42,27 @@ def get_crop(image, crop_name, size=None, attribution=None, exact_size=False):
     values so `exact_size` gives you access to the actual size of an image.
     """
 
-    if not image:
+    if not image or not image.related_object:
         return
 
     if size:
         warnings.warn("The size kwarg is deprecated.", DeprecationWarning)
 
+    if exact_size:
+        warnings.warn("The exact_size kwarg is deprecated.", DeprecationWarning)
+
```

Comment:
We can remove the deprecation warnings if it makes you feel better. Passing size and exact size in have no effect besides to warn that they don't do anything.